packages/manager/modules/pci/src/projects/project/storages/containers/add/add.html

```diff
@@ -14,7 +14,12 @@
         data-on-submit="$ctrl.onRegionChange()"
         data-editable="!$ctrl.isLoading"
         name="storage_container_add_region">
+        <oui-message type="error"
```

Comment:
I want to say `data-` but I'm not sure anymore 😄

Reply:
added for concistency